### DISCUSSION TITLE
[Feature] Adding the option to remove an unnecessary part of the name

### DISCUSSION THREAD

**Discussion by XTorLukas** [sunecz/Media-Downloader#16] [Ideas]
- I would like to see the option to set the automatic discouragement of unnecessary parts in the filename
- Maybe the settings would have a field for entering the patter separated by `;`

## Example remove: ` - iVysílání  Česká televize`
- `NAME - iVysílání  Česká televize.EXT` --> `NAME.EXT`

**Comment by sunecz** [comment]
Hello,

this text is probably part of the title in Linked Data on the video page, which is the source from which the program gets the different parts of the final title of a media.

Can you please share a link to a video that has this unnecessary part in its title? Thank you!

**Reply by XTorLukas** [reply]
Hello,

these are all links from iVysilani, e.g. https://www.ceskatelevize.cz/porady/1003204835-pelisky/

You could just add some input field for the filter pattern, separated by `;` for example, and then just apply `.replace(patternA, "")` to the name. Or just remove from ivysilani links.

But using a matching pattern would make it universal and user configurable. Thank you
